umi_gripper/umi_gripper.xml

```diff
@@ -0,0 +1,141 @@
+    <default class="umi_model">
+      <!-- Joint Defaults -->
+      <joint frictionloss="0.1" armature="0.1" />
+
+      <!-- Gripper Joints -->
+      <default class="gripper_slide">
+        <joint type="slide" damping="0.1" range="-1.2 1.2" />
+        <position kp="100" dampratio="1" ctrlrange="-1.2 1.2" />
+      </default>
+      <default class="gripper_hinge">
+        <joint type="hinge" damping="10.1" range="-31.415 31.415" />
+        <position kp="10" dampratio="0.1" ctrlrange="-3.1415 3.1415" />
+      </default>
+
+      <!-- Finger Joints -->
+      <default class="finger_slide">
+        <joint type="slide" stiffness="100" damping="10.1" range="0 0.05" />
+      </default>
+
+      <!-- Actuator Defaults -->
+      <position kp="1000" />
+      <default class="finger_actuator">
+        <position ctrlrange="0 0.05" />
+      </default>
+      <default class="gripper_actuator">
+        <position kp="100" dampratio="1" ctrlrange="-1.2 1.2" />
+      </default>
+      <default class="gripper_rot_actuator">
+        <position kp="10" dampratio="0.1" ctrlrange="-3.1415 3.1415" />
+      </default>
+
+      <!-- Geom Defaults -->
+      <default class="marker">
+        <geom type="box" size="0.009 0.009 0.00001" euler="0 0 0" rgba="1 1 1 1" contype="0" conaffinity="0" mass="0.0" />
+      </default>
+      <default class="finger_holder_geom">
+        <geom type="mesh" contype="0" conaffinity="0" group="1" density="0" rgba="1 1 1 1" />
+      </default>
+
+    </default>
+  </default>
+
+  <worldbody>
+
+    <body pos='0 0 0' euler="-1.571 0 0">
+
+      <body name="camera" pos="0 -0.092 0.005" euler="3.14 0 0.0">
+        <camera name="top_camera" mode="fixed" fovy="145" />
+      </body>
+
+      <!-- Positional Joints -->
+      <joint name="gripper_joint_x" axis="1 0 0" class="gripper_slide" />
+      <joint name="gripper_joint_y" axis="0 0 1" class="gripper_slide" />
+      <joint name="gripper_joint_z" axis="0 -1 0" class="gripper_slide" />
+
+      <!-- Rotational Joints -->
+      <joint name="gripper_joint_rx" axis="1 0 0" class="gripper_hinge" />
+      <joint name="gripper_joint_ry" axis="0 0 1" class="gripper_hinge" />
+      <joint name="gripper_joint_rz" axis="0 -1 0" class="gripper_hinge" />
+
+      <!-- Mirrors -->
+      <geom name="left_mirror" type="box" pos="-0.069 -0.057 0.04" size="0.03 0.0187 0.0001" euler="0 1.19 0"
+        rgba="0 0 0 1" material="mirror" contype="0" conaffinity="0" />
+      <geom name="right_mirror" type="box" pos="0.069 -0.057 0.04" size="0.03 0.0187 0.0001" euler="0 -1.19 0"
+        rgba="0 0 0 1" material="mirror" contype="0" conaffinity="0" />
+
+      <!-- Base Link -->
+      <geom type="mesh" rgba="1 1 1 1" mesh="base_link" />
+
+      <!-- GoPro Camera -->
+      <geom pos="0.01965 -0.07875 -0.0187" quat="0 0 -0.707108 0.707105" type="mesh" rgba="0.2 0.2 0.2 1" mesh="gopro" />
+
+      <!-- Left Finger -->
+      <body name="finger_holder_left" quat="0.707105 0.707108 0 0">
```

Comment:
Nit: Use unnormalized quats and let MuJoCo deal with normalization.

`quat=".7 .7 0 0"` --> `quat="1 1 0 0"`